src/spikeinterface/sorters/tests/test_runsorter.py

```diff
@@ -19,6 +21,10 @@ def generate_recording():
     return _generate_recording()
 
 
+@pytest.mark.xfail(
```

Comment:
@alejoe91, was the equality sign in python meant to be >= instead of just larger?